python/tests/datasources/test_acs_population.py

```diff
@@ -56,6 +60,34 @@ def get_sex_by_age_county_value_as_df(concept):
         os.path.join(TEST_DIR, filename), dtype={'county_fips': str}).reset_index(drop=True)
 
 
+# We export this function for use in other packages so it needs its own tests
+def testGenerateNationalDatasetRace():
+    state_df = pd.read_csv(os.path.join(TEST_DIR, 'national', 'state_by_race.csv'), dtype={'state_fips': str})
+    expected_df = pd.read_csv(os.path.join(TEST_DIR, 'national', 'national_by_race.csv'), dtype={'state_fips': str})
+    states_to_include = {'01', '06'}
+
+    national_df = generate_national_dataset(state_df, states_to_include, 'race')
+    assert_frame_equal(national_df, expected_df, check_like=True)
+
+
+def testGenerateNationalDatasetSex():
+    state_df = pd.read_csv(os.path.join(TEST_DIR, 'national', 'state_by_sex.csv'), dtype={'state_fips': str})
+    expected_df = pd.read_csv(os.path.join(TEST_DIR, 'national', 'national_by_sex.csv'), dtype={'state_fips': str})
+    states_to_include = {'01', '06'}
+
+    national_df = generate_national_dataset(state_df, states_to_include, 'sex')
+    assert_frame_equal(national_df, expected_df, check_like=True)
```

Comment:
in some of the other files we addressed #1436 by using
```
assert set(national_df.columns) == set(expected_df.columns)
```

not sure it that would be helpful here?